lib/versions.js

```diff
@@ -0,0 +1,81 @@
+const chalk = require("chalk");
+
+/**
+ * Prints the versions overview
+ */
+class Versions
+{
+    /**
+     *
+     * @param {Object.<string,string>} packages
+     */
+    constructor (packages)
+    {
+        /**
+         * Mapping of package -> color to print the version number
+         * @type {Object<string, string>}
+         */
+        this.packages = packages;
+    }
+
+
+    /**
+     * Prints the version overview
+     */
+    print ()
+    {
+        const maxLength = this.getMaxPackageNameLength();
+
+        for (const packageName in this.packages)
```

Comment:
Same comment as above: Is `for..of` an option here?

Reply:
I also need the key here.